### PR TITLE
Add Annotation component

### PR DESCRIPTION
**Pull request by anajavi**
Adds basic annotation support.

### CODE REVIEW

**Comment by whawker** [comment]
This all seems reasonable, would be worth adding an example with annotations I think.

File: packages/react-jsx-highcharts/src/utils/warnings.js [hunk 1]
```diff
@@ -51,6 +53,7 @@ const moduleToVarName = {
 };
 
 const moduleToSeriesTypes = {
+  annotations: ['annotations'],
```
Comment (whawker) [inline packages/react-jsx-highcharts/src/utils/warnings.js]: Maybe worth renaming this var to `moduleToFeatureMap` ?

Comment (anajavi) [reply packages/react-jsx-highcharts/src/utils/warnings.js]: I couldn't figure good name for it, so thank you!

**Comment by anajavi** [comment]
Added an example.

**Comment by anajavi** [comment]
alpha release or wait for highcharts 7?

**Comment by anajavi** [comment]
Maybe @knod from #182 would like to test alpha?

**Comment by whawker** [comment]
I would suggest an alpha release, are you happy with the stability of the Lodash removal stuff you've been working on?

**Comment by anajavi** [comment]
I see you already fixed null reference from one of them, sorry about that.

I think I will check them one more time.

**Comment by knod** [comment]
Sure, if it'd be helpful I can try it out. I can just clone this lib and try it out from there if it's not in an npm-able alpha yet.

**Comment by knod** [comment]
Sure, if it'd be helpful I can try it out. I can just clone this lib and try it out from there if it's not in an npm-able alpha yet. Got any specific questions you want me to focus on?

**Comment by anajavi** [comment]
@whawker I added some tests for the lodash changes. Now I think it is ready to be released.

@knod there is an example at https://github.com/whawker/react-jsx-highcharts/blob/master/examples/AreaWithAnnotations/App.js
I got no specific questions, just want to know if it is easier to work with.

**Comment by anajavi** [comment]
Works for me. The first render is now very fast.

**Pull request merged**